### PR TITLE
Make date dialogue box

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

File: instat/dlgMakeDate.vb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    Private Sub dlgMakeDate_Load(sender As Object, e As EventArgs) Handles MyBase.Load
+
+    End Sub
+    Private Sub InitialiseDialog()
+        ucrReceiverYear.Selector = ucrSeclectorMakeDate
+        ucrReceiverMonth.Selector = ucrSeclectorMakeDate
+        ucrReceiverDay.Selector = ucrSeclectorMakeDate
+        ucrReceiverYear.SetIncludedDataTypes({"numeric", "factor"})
+        ucrReceiverMonth.SetIncludedDataTypes({"numeric", "factor"})
+        ucrReceiverDay.SetIncludedDataTypes({"numeric", "factor"})
```
Comment (dannyparsons) [inline instat/dlgMakeDate.vb]: I don't think we should have a restriction here. Why not character columns?

**Comment by stevenndungu** [comment]
I had done that because Roger had suggested that the selector was to allow
either numeric or factor columns only. But now that will be using as.Date
it fine not using any restriction.

**Comment by dannyparsons** [comment]
ok great

**Comment by dannyparsons** [comment]
@stevenndungu There is a conflict here with the vbproj file. Can you look into this with @Lunalo. Let me know if you need help.

Generally when you are creating a new dialog, you should create the dialog and send a pull request quickly to avoid conflicts in vbproj. Then you can create a new branch to do the design and add code once it's been merged.

**Pull request closed without merging**